src/test/kotlin/no/liflig/properties/PropertiesLoaderTest.kt

```diff
@@ -0,0 +1,83 @@
+package no.liflig.properties
+
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import java.util.Properties
+
+// By default no properties files will match the default names and be loaded
```

Comment:
```suggestion
// The properties files in this project is named so that by default no files
// matches the conventions (meaning no properties gets loaded by default).
// We override file names in tests to simulate environments matching a specific set of files.
```